### PR TITLE
moved download button to bottom bar

### PR DESCRIPTION
**Pull request by coltonoscopy**
Per Issue #106

### CODE REVIEW

File: src/components/download-links/style.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 download-links {
   position: relative;
-  margin-left: auto;
+  margin-left: 0;
   display: flex;
+  border-left: 0 !important;
```
Comment (dmalan) [inline src/components/download-links/style.scss]: Why the `!important`?

File: src/components/video-controls/index.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import NextChapterButton from '../next-chapter-button';
 import ProgressTimer from '../progress-timer';
 import PlaybackRates from '../playback-rates';
 import CaptionsButton from '../captions-button';
+import DownloadLinks from '../download-links';
```
Comment (dmalan) [inline src/components/video-controls/index.js]: How come this wasn't there before?

File: src/components/download-links/style.scss (hunk 1 above)
Comment (coltonoscopy) [inline src/components/download-links/style.scss]: I took from another component that had that, which Luke had written in; not sure what the use case was, but didn't want to break by deviating!

File: src/components/video-controls/index.js (hunk 2 above)
Comment (coltonoscopy) [inline src/components/video-controls/index.js]: The `VideoControls` component didn't include `DownloadLinks` before (was in `index.js`)!

File: src/components/download-links/style.scss (hunk 1 above)
Comment (coltonoscopy) [inline src/components/download-links/style.scss]: Actually.... this is completely unnecessary, was fixed by `margin-left: 0` and was an experiment. Removing with another incoming commit!

**Comment by dmalan** [comment]
Ah, gotcha.

**Comment by dmalan** [comment]
:shipit:

**Pull request merged**